soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -287,6 +288,172 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
     }
 }
 
+std::vector<RandomizerCheck> shopItemRandomizerChecks = {
+    RC_KF_SHOP_ITEM_1,
+    RC_KF_SHOP_ITEM_2,
+    RC_KF_SHOP_ITEM_3,
+    RC_KF_SHOP_ITEM_4,
+    RC_KF_SHOP_ITEM_5,
+    RC_KF_SHOP_ITEM_6,
+    RC_KF_SHOP_ITEM_7,
+    RC_KF_SHOP_ITEM_8,
+    RC_GC_SHOP_ITEM_1,
+    RC_GC_SHOP_ITEM_2,
+    RC_GC_SHOP_ITEM_3,
+    RC_GC_SHOP_ITEM_4,
+    RC_GC_SHOP_ITEM_5,
+    RC_GC_SHOP_ITEM_6,
+    RC_GC_SHOP_ITEM_7,
+    RC_GC_SHOP_ITEM_8,
+    RC_ZD_SHOP_ITEM_1,
+    RC_ZD_SHOP_ITEM_2,
+    RC_ZD_SHOP_ITEM_3,
+    RC_ZD_SHOP_ITEM_4,
+    RC_ZD_SHOP_ITEM_5,
+    RC_ZD_SHOP_ITEM_6,
+    RC_ZD_SHOP_ITEM_7,
+    RC_ZD_SHOP_ITEM_8,
+    RC_KAK_BAZAAR_ITEM_1,
+    RC_KAK_BAZAAR_ITEM_2,
+    RC_KAK_BAZAAR_ITEM_3,
+    RC_KAK_BAZAAR_ITEM_4,
+    RC_KAK_BAZAAR_ITEM_5,
+    RC_KAK_BAZAAR_ITEM_6,
+    RC_KAK_BAZAAR_ITEM_7,
+    RC_KAK_BAZAAR_ITEM_8,
+    RC_KAK_POTION_SHOP_ITEM_1,
+    RC_KAK_POTION_SHOP_ITEM_2,
+    RC_KAK_POTION_SHOP_ITEM_3,
+    RC_KAK_POTION_SHOP_ITEM_4,
+    RC_KAK_POTION_SHOP_ITEM_5,
+    RC_KAK_POTION_SHOP_ITEM_6,
+    RC_KAK_POTION_SHOP_ITEM_7,
+    RC_KAK_POTION_SHOP_ITEM_8,
+    RC_MARKET_BAZAAR_ITEM_1,
+    RC_MARKET_BAZAAR_ITEM_2,
+    RC_MARKET_BAZAAR_ITEM_3,
+    RC_MARKET_BAZAAR_ITEM_4,
+    RC_MARKET_BAZAAR_ITEM_5,
+    RC_MARKET_BAZAAR_ITEM_6,
+    RC_MARKET_BAZAAR_ITEM_7,
+    RC_MARKET_BAZAAR_ITEM_8,
+    RC_MARKET_POTION_SHOP_ITEM_1,
+    RC_MARKET_POTION_SHOP_ITEM_2,
+    RC_MARKET_POTION_SHOP_ITEM_3,
+    RC_MARKET_POTION_SHOP_ITEM_4,
+    RC_MARKET_POTION_SHOP_ITEM_5,
+    RC_MARKET_POTION_SHOP_ITEM_6,
+    RC_MARKET_POTION_SHOP_ITEM_7,
+    RC_MARKET_POTION_SHOP_ITEM_8,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_1,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_2,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_3,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_4,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_5,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_6,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_7,
+    RC_MARKET_BOMBCHU_SHOP_ITEM_8,
+};
+
+// Reference soh/src/overlays/actors/ovl_En_GirlA/z_en_girla.h
+std::unordered_map<RandomizerGet, int32_t> randomizerGetToEnGirlShopItem = {
```

Comment:
I would like to use the `SI_` enum from z_en_girla.h but it doesn't seem to be available here, can I make it available?